docs/resources/bms_instance.md

```diff
@@ -189,8 +189,14 @@ The following arguments are supported:
 * `auto_renew` - (Optional, String) Specifies whether auto renew is enabled. Valid values are "true" and "
   false", defaults to *false*.
 
-* `agency_name` - (Optional, String, ForceNew) Specifies the IAM agency name which is created on IAM to provide
-  temporary credentials for BMS to access cloud services. Changing this creates a new instance.
+* `agency_name` - (Optional, String) Specifies the IAM agency name which is created on IAM to provide
+  temporary credentials for BMS to access cloud services. User can define the value.
+
+* `metadata` - (Optional, Map) Specifies the user-defined metadata key-value pair. If you do not specify any
+  key-value pair, metadata will not be updated.
```

Comment:
why should we add `If you do not specify any key-value pair, metadata will not be updated` ?